From c1bdc61725caed0d6ae0085f41102b2abf77908a Mon Sep 17 00:00:00 2001
From: Xilin Wu <sophon@radxa.com>
Date: Tue, 21 Apr 2026 23:47:30 +0800
Subject: [PATCH 073/268] drm/bridge: simple: add the Chrontel CH7218A
 DP-to-HDMI bridge

The Chrontel CH7218A is a transparent Displayport to HDMI bridge.

Signed-off-by: Xilin Wu <sophon@radxa.com>
---
 drivers/gpu/drm/bridge/simple-bridge.c | 5 +++++
 1 file changed, 5 insertions(+)

--- a/drivers/gpu/drm/bridge/simple-bridge.c
+++ b/drivers/gpu/drm/bridge/simple-bridge.c
@@ -272,6 +272,11 @@ static const struct of_device_id simple_
 			.connector_type = DRM_MODE_CONNECTOR_HDMIA,
 		},
 	}, {
+		.compatible = "chrontel,ch7218a",
+		.data = &(const struct simple_bridge_info) {
+			.connector_type = DRM_MODE_CONNECTOR_HDMIA,
+		},
+	}, {
 		.compatible = "ti,opa362",
 		.data = &(const struct simple_bridge_info) {
 			.connector_type = DRM_MODE_CONNECTOR_Composite,
